assets/src/stories-editor/components/media-inserter/index.js

```diff
@@ -109,6 +113,17 @@ const applyWithSelect = withSelect( ( select ) => {
 	const { canInsertBlockType, getBlockListSettings } = select( 'core/block-editor' );
 	const { isReordering } = select( 'amp/story' );
 
+	const getCurrentPageBackgroundType = () => {
+		const clientId = getCurrentPage();
+		const block = select( 'core/block-editor' ).getBlock( clientId );
+		if ( ! block ) {
+			return '';
+		}
+	
+		const mediaType = ( block.attributes.mediaType ) ? block.attributes.mediaType : '';
+		return mediaType;
```

Comment:
This could just be `return ( block.attributes.mediaType ) ? block.attributes.mediaType : '';`

Reply:
I personally find it easy to read as is. I don't love having logic in the return statement, for readability. But I don't feel strongly about this.

Reply:
Fixed in https://github.com/ampproject/amp-wp/pull/3109/commits/c053210919f8d02ec8da3b1a5f65ada5d6129891